### ISSUE TITLE
Possibility to set topic retention attributes when creating them from ksqldb

### ISSUE THREAD

**Issue by dberardo-com** [confluentinc/ksql#9151]
Not all topics need to use the default retention settings (policies, bytes, .ms) when created from KSQLDB queries.

it would be extremely helpful if one could pass more topic configuration parameters in the queries:

CREATE STREAM/TABLE .... WITH (PARTITIONS=x, .... RETENTION_MS, RETENTION_POLICY, ... others ...)

**Comment by chinvunified** [comment]
Yeah, we need this feature. We have hourly data collected job and don't want to keep tables data too long (likely 2 weeks) to save disk space.

**Comment by Adityaprinc** [comment]
I have usability where i create topics on runtime and their retension time is going to be different. I am missing this feature badly.

**Comment by itsmestefanjay** [comment]
Hello @agavra, are there any news on this issue?
